src/event/service.js

```diff
@@ -38,6 +38,33 @@ angular
     /* Cities
     /*==============================================================*/
 
+   /**
+    * List of events for the provided city. 
+    * Returns a promise. Supports additional 'options'.
+    *
+    * @method listEventsForCity
+    * @param {Object} options 
+    * @param {String} options.cityId - ID of a city
+    * @param {String} options.fields - CSV of fields to return
+    * @param {String} options.start  - unix start date for events
+    * @param {String} options.end    - unix end date for events
+    */
+
+    EventService.prototype.listEventsForCity = function(options) {
```

Comment:
There's a couple places in web that still use ```listForCity```. I could change them over. Simply a change from using a callback, to using a promise. I just didn't want to deprecate that function in the case that some web of our other projects might still reference it...